src/main/java/org/apache/mesos/offer/TaskUtils.java

```diff
@@ -309,12 +310,49 @@ public static String getTaskType(TaskInfo taskInfo) throws InvalidProtocolBuffer
         }
     }
 
-    public static CommandInfo getCommand(TaskInfo taskInfo) throws InvalidProtocolBufferException {
-        return Protos.CommandInfo.parseFrom(taskInfo.getData());
+    public static TaskInfo serializeTaskInfo(TaskInfo taskInfo) {
```

Comment:
@nickbp and @mrbrowning:  I went with pack/unpack and added some comments.  It's not a serialization of ExecutorInfo.  The fact that there's an ExecutorInfo involved at all is an abomination.